### PR TITLE
Fix SMPTE 291 exactframerate field

### PR DESCRIPTION
**Pull request by garethsb**
Unit tests + basically same fix for #344 as proposed by @cmaberto in #345.

### CODE REVIEW

**Comment by garethsb** [comment]
New unit tests found another simple bug so I've fixed that too 🤦‍♂️

File: Development/nmos/test/sdp_utils_test.cpp [hunk 1]
```diff
@@ -498,3 +498,332 @@ BST_TEST_CASE(testSdpTransportParamsMulticast)
...
+                { U("height"), U("6666") },
+                { U("exactframerate"), U("123") },
+                { U("interlace"), {} },
+                { U("segmented"), {} },
+                { U("TCS"), U("ST2115LOGS3") },
+                { U("colorimetry"), U("BT2100") },
+                { U("RANGE"), U("FULLPROTECT") },
+                { U("PAR"), U("12:11") },
+                { U("PM"), U("2110BPM") },
+                { U("SSN"), U("ST2110-20:2022") },
```
Comment (jonathan-r-thorpe) [inline Development/nmos/test/sdp_utils_test.cpp:601]:
```suggestion
                { U("SSN"), U("ST2110-20:2021") },
```

File: Development/nmos/test/sdp_utils_test.cpp [hunk 2]
```diff
@@ -498,3 +498,332 @@ BST_TEST_CASE(testSdpTransportParamsMulticast)
...
+                U("smpte291"),
+                90000
+            },
+            {
+                { U("DID_SDID"), U("{0xAB,0xCD}") },
+                { U("DID_SDID"), U("{0xEF,0x01}") },
+                { U("VPID_Code"), U("132") },
+                { U("exactframerate"), U("60000/1001") },
+                { U("TM"), U("CTM") },
+                { U("SSN"), U("ST2110-40:2022") },
```
Comment (jonathan-r-thorpe) [inline Development/nmos/test/sdp_utils_test.cpp:753]:
```suggestion
                { U("SSN"), U("ST2110-40:2021") },
```

**Comment by garethsb** [comment]
@jonathan-r-thorpe I believe this is now failing just because #351 isn't merged yet... so I guess the proper procedure is to merge #351 into *master*, then merge *master* into this PR branch, check the CI results and then merge this to master. Sigh.

File: Development/nmos/test/sdp_utils_test.cpp [hunk 3]
```diff
@@ -498,3 +498,332 @@ BST_TEST_CASE(testSdpTransportParamsMulticast)
...
+                { U("TSMODE"), U("SAMP") },
+                { U("TSDELAY"), U("82") }
+            }
+        },
+        {
+            { { 0xAB, 0xCD }, { 0xEF, 0x01 } },
+            nmos::vpid_codes::vpid_1_5Gbps_720_line,
+            nmos::rates::rate59_94,
+            sdp::transmission_models::compatible,
+            sdp::smpte_standard_numbers::ST2110_40_2022,
```
Comment (jonathan-r-thorpe) [inline Development/nmos/test/sdp_utils_test.cpp:764]:
```suggestion
            sdp::smpte_standard_numbers::ST2110_40_2023,
```

**Comment by jonathan-r-thorpe** [comment]
Verified with the updated [SDPoker](https://github.com/AMWA-TV/sdpoker/pull/38). LGTM

**Review by jonathan-r-thorpe — APPROVED** [review]
LGTM

**Pull request merged**